specification/batch/Azure.Batch/routes.tsp

```diff
@@ -41,17 +48,195 @@ interface Applications {
     API.
     """)
   @route("/applications/{applicationId}")
-  @clientName("getApplicationInternal", "java")
   getApplication is ReadOperation<
     {
-      @doc("The ID of the Application")
-      @path
-      applicationId: string;
+      ...CommonParameters;
+      ...GetApplicationRequiredParameters;
     },
     BatchApplication
   >;
 }
 
+// Pool Parameter Aliases
+
+alias ListPoolUsageMetricsParameters = {
```

Comment:
Not sure if its a big deal or not but all of the other alias for parameters live in the common.tsp. Seems like we should have them in one place, either here or common

Reply:
Just pushed a commit to move all the aliases in routes.tsp to common.tsp